data/json/vehicleparts/boards.json

```diff
@@ -1,21 +1,40 @@
 [
   {
-    "abstract": "board",
+    "abstract": "board_abstract",
     "type": "vehicle_part",
-    "name": { "str": "board" },
     "item": "sheet_metal",
     "location": "center",
+    "standard_symbols": true,
+    "symbols": {
+      "horizontal_front": "T",
+      "horizontal_rear": "B",
+      "horizontal_2_front": "=",
+      "horizontal_2_rear": "=",
+      "vertical_left": "j",
+      "vertical_right": "j",
+      "vertical_T_left": "[",
+      "vertical_T_right": "]",
+      "vertical_2_left": "H",
+      "vertical_2_right": "H"
```

Comment:
At least in the current JSON, board only has horizontal front/rear and vertical left/right, as well as the corners nw/ne/sw/se. Halfboard has the h/v 2 and v_T versions as well as halfboard_cover.

Reply:
My preference would be to define all of them once and leave most of them unused.

Also, there needs to be some way to distinguish between left/right variants in the install menu.

Reply:
If they're defined though, they can be installed by players, which means we still need those big piles of art.

Reply:
There's an automatic fallback to the base part, and from there to its looks_like, so I don't see the hard requirement for big piles of art.

Reply:
Can the automatic fallback go back to the most *similar* part? If someone installs vertical_2_left, it should look like vertical_left, not the base part which normally looks like _horizontal.

Reply:
let's get this working as is, then come back and figure out how to make looks_like work with it.